bin/cf_inspect

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+action () {
+    local shell_is_zsh="$( [ -z "${ZSH_VERSION}" ] && echo "false" || echo "true" )"
+    local this_file="$( ${shell_is_zsh} && echo "${(%):-%x}" || echo "${BASH_SOURCE[0]}" )"
+    local this_dir="$( cd "$( dirname "${this_file}" )" && pwd )"
+
+    [ "$#" -eq 0 ] && {
+        echo "ERROR: at least one file must be provided"
+        return 1
+    }
```

Comment:
I would actually leave this check in the shell script in this particular case because the `cf_inspect` script calls `ipython -i`, so letting the ArgumentParser raise the exception on the Python side results in a long traceback that can make it difficult to understand what went wrong and also leaves the interactive shell prompt open which then has to be closed by hand.